### ISSUE TITLE
RNA-Seq by Example, error in chapter 8

### ISSUE THREAD

**Issue by darkocerdic** [biostars/biostar-handbook-issues#126]
- Chapter 8: **Which method is the best?**
- Section: **Which genes are found only with C (edger) and not A (deseq1)**

The command:
```
bio common A C --uniq1
```
produces:
```
BAB-1500000-SAME-1
BCB-93750-SAME-1
```
instead of:
```
CAC-2250000-DOWN-0.67
CHC-8789-DOWN-0.67
CIC-4395-DOWN-0.67
CPC-275-DOWN-0.67
DKD-2930-DOWN-0.5
DMD-732-DOWN-0.5
```

The correct command for getting the wanted output is:
```
bio common A C --uniq2
```

So it should be `--uniq2` instead of `--uniq1`.

![8__Which_method_is_the_best____RNA-Seq_by_Example](https://user-images.githubusercontent.com/998371/90386336-b11d1700-e084-11ea-9826-4865c66a6717.png)

**Comment by ialbert** [comment]
You are correct, our apologies for the typo, it will be fixed on the next update. 

This issue will be closed once the correction is made.
